README.md

```diff
@@ -76,10 +68,6 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
     # good
     def self.create_translation(phrase_id,
                                 phrase_key,
-                                target_locale,
-                                value,
-                                user_id,
-                                do_xss_check,
```

Comment:
This one looks funky because after removing these params, they would all fit on one line.